pom.xml

```diff
@@ -95,11 +103,16 @@
                 <version>1.1.4c</version>
                 <scope>provided</scope>
             </dependency>
+			<dependency>
+				<groupId>javax.annotation</groupId>
+				<artifactId>javax.annotation-api</artifactId>
```

Comment:
Is there a reason why you keep forgetting the scope?

Reply:
What scope should it be? I tested with the 'provided' and it is giving me additional errors that's why I removed it.

Reply:
You do not want this dependency to get compiled with the module because it will cause chaos for those using in on lower versions of Java which already have this api.

Reply:
I updated the PR